services/provider/server/server.go

```diff
@@ -74,6 +77,12 @@ const (
 	clientInfoRbdClientProfileKey    = "csiop-rbd-client-profile"
 )
 
+var (
+	lastKnownInstalledCSVname string
```

Comment:
As of our standard, we only uppercase the first letter of an any word including abbreviation 
```suggestion
	lastKnownInstalledCsvName string
```